secrets_management/javascript/sdk/order-processor/index.js

```diff
@@ -0,0 +1,14 @@
+import { DaprClient, CommunicationProtocolEnum } from 'dapr-client'; 
+
+const DAPR_HOST = process.env.DAPR_HOST || "http://localhost";
+const DAPR_HTTP_PORT = process.env.DAPR_HTTP_PORT || "3500";
+const DAPR_SECRET_STORE = "localsecretstore";
+const SECRET_NAME = "secret";
+
+async function main() {
+    const client = new DaprClient(DAPR_HOST, DAPR_HTTP_PORT, CommunicationProtocolEnum.HTTP);
+    var secret = await client.secret.get(DAPR_SECRET_STORE, SECRET_NAME);
```

Comment:
cool you used the modern await syntax here!  Just if you use `const` for variables please use consistently throughout instead of `var`.  This would change to:

`const secret = await client.secret.get(DAPR_SECRET_STORE, SECRET_NAME);`